mk/spksrc.spk.mk

```diff
@@ -347,7 +347,7 @@ endif
 
 KERNEL_REQUIRED = $(MAKE) kernel-required
 ifeq ($(strip $(KERNEL_REQUIRED)),)
-ALL_ACTION = $(sort $(basename $(subst -,.,$(basename $(subst .,,$(ARCHS_NO_KRNLSUPP))))))
+ALL_ACTION = $(sort $(basename $(subst -,.,$(basename $(subst .,,$(ARCHS_WITH_KERNEL_SUPPORT))))))
```

Comment:
I suggest once the patchset is in a readiness state that we go over the documentation for all the new variables that needs to be adjusted and/or removed/added.

Reply:
yes indeed. First I renamed ARCHS_NO_KRNLSUPP to ARCHS_WITH_KERNEL_SUPPORT to give it a correct naming.

Reply:
This variable was framework internal and not documented in any wiki page.

Reply:
And could potentially be reworked with the TC_* requirement makefile...